### PR TITLE
[MC-156] Front demo bot

### PR DESCRIPTION
**Pull request by dcecile**
### Fixes
- [MC-156](https://meyaai.atlassian.net/browse/MC-156)

### Tasks
- [x] Set up restart flow
- [x] Set up Dialogflow
- [x] [Orb] Authenticate using orb user ID
- [x] Set up FAQs
- [x] Create Front contact if not created/linked
- [x] Triage general question
- [x] Configure correct "agent assigned" status message
- [x] Send CSAT when conversation is assigned to bot
- [x] Handle add tag event (e.g. high priority)
- [x] Export Dialogflow
- [x] Customize agent attribution (e.g. initials name, SVG avatar)
- [x] Enable sensitive data
- [x] Triage missing points with dropdown select & receipt upload issue
- [x] Sync app user data to Front contact
- [x] Help agent collect data using agent command
- [x] Tell user if no agents active
- [x] Remove or solve TODOs
- [x] Set up GitHub Action

### Loom video
https://www.loom.com/share/6fa59e5082ce426ca403e6e421d85f67

### CODE REVIEW

**Comment by jgbmattos** [comment]
LGTM!

File: flow/nlu/get_intents.yaml [hunk 1]
```diff
@@ -1,45 +1,32 @@
 steps:
   - type: component.nlu.tokenize_text
-    text: (@ flow.result )
+    text: (@ flow.text )
```
Comment (jgbmattos) [inline flow/nlu/get_intents.yaml:3]: This flow seems unused. Is that right?

Also it is very similar to `flow/nlu/inbound.yaml`. You could reuse that code... passing a variable is_inbound for example.

Comment (dcecile) [reply flow/nlu/get_intents.yaml:3]: 👍 I didn't finish the email flows this time (this is related, maybe a bug that it's unused). We can clean it up in the next issue: https://meyaai.atlassian.net/browse/MC-160?focusedCommentId=11835

File: integration/front.yaml [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 api_token: (@ vault.front.api_token )
 bot_teammate_email: (@ vault.front.bot_teammate_email )
 channel_id: (@ vault.front.channel_id )
 unassign_mode: bot
-integration_mode: ticketing
-unlink_thread_status:
-  - trash
+integration_mode: (@ vault.front.mode )
 agent:
   name: first_initial_last_initial
```
Comment (ekalvi) [inline integration/front.yaml]:
```suggestion
  name: first_last_initial
```

**Review by ekalvi — CHANGES_REQUESTED** [review]
LGTM w/ one minor suggestion.

**Review by ekalvi — APPROVED** [review]
✅

**Pull request merged**